common.gypi

```diff
@@ -419,7 +427,7 @@
             'cflags': [ '-m64', '-mminimal-toc' ],
             'ldflags': [ '-m64' ],
           }],
-          [ 'target_arch=="s390x"', {
+          [ 'target_arch=="s390x" and OS!="zos"', {
```

Comment:
Perhaps we should check OS == "linux" instead here, as the options are LoZ specific.